### DISCUSSION TITLE
will you supporting for lapce in the future?

### DISCUSSION THREAD

**Discussion by hi-reeve** [vuejs/language-tools#1915] [Q&A] [answered]
Like in the title, seems like lapce will have a good future, with it's fast performance. will you support it when it is stable enough?

**Comment by johnsoncodehk** [comment, marked as answer]
@xiaoxin-sky just doing this work, please track for lapce updates.

**Reply by xiaoxin-sky** [reply]
The `lapce-vue` has arrived([here](https://github.com/xiaoxin-sky/lapce-vue/releases/tag/v0.0.1)). In the near future, it will be available for download in the lapce plugin market.
